bindings/python/tests/test_compatibility_layer.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import capstone.arm64
+import capstone.sysz_const
+from capstone import *
+import capstone.aarch64
+import capstone.systemz
+from xprint import to_hex
+
+
+AARCH64_CODE = b"\x21\x7c\x02\x9b\x21\x7c\x00\x53\x00\x40\x21\x4b\xe1\x0b\x40\xb9"
+SYSZ_CODE = b"\xed\x00\x00\x00\x00\x1a\x5a\x0f\x1f\xff\xc2\x09\x80\x00\x00\x00\x07\xf7\xeb\x2a\xff\xff\x7f\x57\xe3\x01\xff\xff\x7f\x57\xeb\x00\xf0\x00\x00\x24\xb2\x4f\x00\x78"
+
+
+all_tests = (
+    (CS_ARCH_ARM64, CS_MODE_ARM, AARCH64_CODE, "ARM64"),
+    (CS_ARCH_SYSZ, CS_MODE_BIG_ENDIAN, SYSZ_CODE, "SysZ"),
+)
+
+
+# ## Test class Cs
+def test_class():
+    for arch, mode, code, comment in all_tests:
+        print("*" * 16)
+        print("Platform: %s" % comment)
+        print("Code: %s" % to_hex(code))
+        print("Disasm:")
+
+        try:
+            md = Cs(arch, mode)
+
+            for insn in md.disasm(code, 0x1000):
+                print("0x%x:\t%s\t%s" % (insn.address, insn.mnemonic, insn.op_str))
```

Comment:
Please access at least one detail for each architecture. Just to catch any problems down we might get in the future with the details.